types/urijs/index.d.ts

```diff
@@ -212,7 +212,8 @@ interface URI {
     preventInvalidHostname(val: boolean): URI;
 
     query(): string;
-    query(qry: string | URI.QueryDataMap | ((qryObject: URI.QueryDataMap) => (URI.QueryDataMap | undefined))): URI;
+    // tslint:disable-next-line void-return
+    query(qry: string | URI.QueryDataMap | ((qryObject: URI.QueryDataMap) => (URI.QueryDataMap | void))): URI;
```

Comment:
While this next version is more correct, returning undefined also works (has the same effect as `void`). So I'd say let's keep both.

What do you think?

Reply:
![image](https://user-images.githubusercontent.com/543372/92367593-e6061200-f0f6-11ea-90ad-1a3f002688c9.png)